pkgs/by-name/me/mealie/package.nix

```diff
@@ -55,11 +56,6 @@ pythonpkgs.buildPythonApplication rec {
 
   pythonRelaxDeps = true;
 
-  patches = [
-      # compatibility with openai 1.63.0
-      ./0000_openai_1.63.0.patch
```

Comment:
Please also delete the patch file, not just the mention here

Reply:
I forgot about that, thanks for spotting it!